web-server/v0.4/src/pages/Explore/index.js

```diff
@@ -1,7 +1,7 @@
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
We should consider enforcing the use of a destructured reference to `Component` so we can say `class Explore extends Component {` on line 14 to maintain consistent component definition structure throughout the codebase.

Reply:
Done!